src/app.js

```diff
@@ -9,23 +12,32 @@ class App extends React.Component {
         button: 'get from query',
       },
       query: '["metrosById", [72], ["name"]]',
```

Comment:
let's have a chat today at some point re what it is still outstanding in terms of dependencies that need to be injected